src/components/SquareBox/SquareBox.tsx

```diff
@@ -33,8 +35,18 @@ const styles = StyleSheet.create({
 
 export function SquareBox() {
     const id = React.useId();
-        const [squares, setSquares] = React.useState<Array<string>>([]);
-      
+        const [squares, setSquares] = React.useState<Array<Object>>([]);
+const addCircle=function(){
```

Comment:
Please change the regular function to arrow function - it takes less space what makes the code more readable